telemetry/roles/prometheus_gaudi/tasks/main.yml

```diff
@@ -13,8 +13,11 @@
 # limitations under the License.
 ---
 
-- name: Check if Gaudi Prometheus metric exporter need to be deployed
-  when: hostvars['127.0.0.1']['prometheus_gaudi_support']
+- name: Check if telemetry entry is present in software_config.json
+  when: hostvars['127.0.0.1']['telemetry_entry_present']
```

Comment:
we have change in internal repo, this was part of one of the defect fix
![image](https://github.com/user-attachments/assets/1ead44b4-e889-4803-9d3a-04478e404740)